ui/lib/core/addon/modifiers/code-mirror.js

```diff
@@ -18,58 +18,54 @@ import 'codemirror/mode/ruby/ruby';
 import 'codemirror/mode/javascript/javascript';
 
 export default class CodeMirrorModifier extends Modifier {
-  didInstall() {
-    this._setup();
-  }
-
-  didUpdateArguments() {
-    this._editor.setOption('readOnly', this.args.named.readOnly);
-    if (!this.args.named.content) {
-      return;
-    }
-    if (this._editor.getValue() !== this.args.named.content) {
-      this._editor.setValue(this.args.named.content);
+  modify(element, positionalArgs, namedArgs) {
```

Comment:
Codemirror API changed -- [see commit](https://github.com/hashicorp/vault/pull/21955/commits/115cdf34ac2be4fcc10fdd16f508585e58043894) for context